docs/source/aryn_cloud/gentle_introduction.md

```diff
@@ -0,0 +1,220 @@
+## A Gentle Introduction to the Aryn Partitioning Service 
+You can use the Aryn Partitioning Service to easily chunk and extract data from complex PDFs. The Partitioning Service can extract paragraphs, tables and  images and returns detailed information about the components it has just identified in a JSON object.  The following two sections will walk you through two examples where we segment  PDF documents and extract a table and an image from those documents using the python aryn-sdk.
+
+### Extracting Tables from a PDF
+
+In [this example](https://colab.research.google.com/drive/1ZkJ9clSVVvJsNPRlWXpplfs1U9W4ZSdH?usp=sharing), we’ll use the Partitioning Service to extract the “Supplemental Income” table (shown below) from the 10k financial document  of 3M, and turn it into a pandas dataframe. 
+
+![alt text](3m_supplemental_income.png)
+
+We’ll go through the important code snippets below  to see what’s going on.  (Try out it out in  [colab](https://colab.research.google.com/drive/1ZkJ9clSVVvJsNPRlWXpplfs1U9W4ZSdH?usp=sharing)  yourself! )
+
+
+Let’s focus on the following code that makes a call to the Aryn Partitioning Service: 
+
+```
+import aryn_sdk
+from aryn_sdk.partition import partition_file, tables_to_pandas
+import pandas as pd
+from io import BytesIO
+
+## Make a call to the Aryn Partitioning Service (APS) 
+## param extract_table_structure (boolean): extract tables and their structural content. default: False
+## param use_ocr (boolean): extract text using an OCR model instead of extracting embedded text in PDF. default: False
+## returns: JSON object with elements representing information inside the PDF
+partitioned_file = partition_file(curr_file, aryn_api_key, extract_table_structure=True, use_ocr=True)
+```
+
+If you inspect the partitioned_file variable, you’ll notice that it’s a large JSON object with details about all the components in the PDF (checkout [this page](./aps_output.md) to understand the schema of the returned JSON object in detail).  Below, we highlight  the ‘table’ element that contains the information about the table in the page.
+
+```
```

Comment:
this can prob be ```json

Reply:
I'll fix the python blocks for the pretty-printing. The json blocks are incomplete blocks (the entire JSON doc is huge so I'm just including snippets) so adding ```json is highlighting all the syntax issues lol